Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml

```
@@ -248,6 +248,12 @@
   id: MobAdultSlimesYellowAngry
   suffix: Angry
   components:
+    - type: NFSalvageMobRestrictions
+      despawnIfOffLinkedGrid: false
+      addComponentsOnDeath: []
+      removeComponentsOnDeath:
+      - type: GhostRole
+  
```

This seems like a clumsy fix.  Why is this needed when, for example, snails and mothroaches don't have it?
Add GhostTakeoverAvailable instead.
```suggestion
    - type: GhostTakeoverAvailable # Frontier
```